lumigator/backend/backend/tracking/mlflow.py

```diff
@@ -221,6 +221,7 @@ def get_workflow(self, workflow_id: str) -> WorkflowDetailsResponse:
         # sort the jobs by created_at, with the oldest last
         all_jobs = sorted(all_jobs, key=lambda x: x.info.start_time)
 
+        (loguru.logger.critical(workflow.data.tags),)
```

Comment:
Can you explain what it being inside parenthesis means? I'm not familiar with this syntax.

Reply:
It means that I need to be more careful with my test code :-|
Actually, I believe I incorrectly pasted it without (), which make a valid tuple afaict, and VSCode added parenthesis to make it more explicit that it was a tuple. ANYWAY it has to be removed.